pdf/creator/paragraph.go

```diff
@@ -193,9 +193,8 @@ func (p *Paragraph) SetWidth(width float64) {
 func (p *Paragraph) Width() float64 {
 	if p.enableWrap {
 		return p.wrapWidth
-	} else {
-		return p.getTextWidth() / 1000.0
 	}
+	return p.getTextWidth() / 1000.0
 }
 
 // Height returns the height of the Paragraph. The height is calculated based on the input text and how it is wrapped
```

Comment:
Should end with period as godoc style.

Reply:
Actually it does end with a period (sentence is continued in following line), see
https://github.com/unidoc/unidoc/blob/d5fc69b21b75ee1dcb7ae49da519071434766025/pdf/creator/paragraph.go#L201